prisma/models/ProductToTag.prisma

```diff
@@ -0,0 +1,6 @@
+model ProductToTag {
+  product_id           String  @unique
+  product              Product @relation("product_tag", fields: [product_id], references: [id], onDelete: Cascade)
+  tag_name             String  @unique
```

Comment:
I think that both product_id and tag_name shouldn't be unique because we might have products with multiple tags and tags with multiple products in this table.